ui/js/controllers/main.js

```diff
@@ -341,14 +341,14 @@ treeherderApp.controller('MainCtrl', [
             }],
 
             // Shortcut: select next unclassified failure
-            [['j', 'n'], function() {
+            ['n', function() {
```

Comment:
I can't speak for what the current sheriffs use, but I only ever used j/k and not n/p, so this change would have really messed with my muscle memory. Though happy for them to make the call on this.